### PR TITLE
Translations update from Hosted Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Mbin/Mbin](https://hosted.weblate.org/projects/mbin/mbin/).

Current translation status:

Weblate translation status

### CODE REVIEW

**Review by nobodyatroot — APPROVED** [review]

File: translations/messages.ru.yaml [hunk 1]
```diff
@@ -376,8 +376,7 @@ kbin_intro_title: Посмотреть Федиверс
 kbin_intro_desc: является децентрализованной платформой для агрегирования контента
   и микроблогинга, которая работает в сети Fediverse.
 kbin_promo_title: Создайте ваш инстанс
-kbin_promo_desc: '%Начальная ссылка%Клонировать репозиторий%Конечная ссылка% и развивайте
-  Федиверс'
+kbin_promo_desc: 'перейдите по ссылке и начните свой проект'
```
Comment (e-five256) [inline translations/messages.ru.yaml:379]: It looks like this lost the link, I think there's some confusion on the tokens. 

If you take https://github.com/MbinOrg/mbin/blob/main/translations/messages.nl.yaml#L379 for instance, it has `%link_start%` and `%link_end%` around where the link is. These aren't meant to be translated, they are replacement tokens used to insert the proper HTML

Comment (nobodyatroot) [reply translations/messages.ru.yaml:379]: oops my bad on the approval, @gregscoor can you address this?

Comment (doesnmisclown) [reply translations/messages.ru.yaml:379]: madved says if we insert %begin_link% in this string weblate warnings about uncompleted translate

Comment (gregscoor) [reply translations/messages.ru.yaml:379]: as for me, so I would remove this line from the translation

**Pull request merged**